modules/common/src/Storage/SelectFactory.php

```diff
@@ -152,6 +150,10 @@ private function normalizeProperty(mixed $property): object {
     }
     // Throw exception if obviously unsafe property name.
     self::safeProperty($property->property);
+    return $property;
+  }
+
+  private function sanitizeProperty(object $property) {
```

Comment:
@janette I think splitting this function potentially causing the `$alias` error but I am not sure. It is one place to debug.